.travis.yml

```diff
@@ -21,10 +20,14 @@ env:
 
 matrix:
   fast_finish: true
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION="2.8.*"
```

Comment:
3.0 is installed by default, as it's the last version. So it would duplicate 5.6 run